### PR TITLE
Style details

### PR DESCRIPTION
**Pull request by jszwedko**
Initial styling of details page

Will iterate, but want to start getting feedback on structure.

* Add moment.js for date formatting
* Add libphonenumber-js for phone number formatting
* Update Citygram.getEvent to return GeoJSON, making it easier to directly use with maps
* Reorganize template structure to match design, adding in conditionals to handle lack of data

TODOs:
* Figure out how to refactor out the shared map elements with NoticesMap component.
* Pull out the bits that reference "permit" and generalize for other event types.

Thoughts on both approaches for TODOs welcome!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
     "html-webpack-plugin": "^3.2.0",
     "leaflet": "^1.3.1",
     "leaflet-defaulticon-compatibility": "^0.1.1",
+    "libphonenumber-js": "^1.4.2",
+    "moment": "^2.22.2",
```
Comment (aekong) [inline package.json:17]: i ❤️ moment.js

Comment (jszwedko) [reply package.json:17]: It's an elegant library

**Comment by aekong** [comment]
this looks ok to me - but this is my comment in the context of not having any vue experience.

regarding the map, would it be possible to create a service that can be shared across components?

**Review by aekong — APPROVED** [review]

**Comment by jszwedko** [comment]
Yeah, they use mixins (https://vuejs.org/v2/guide/mixins.html) to share functionality. I'd started down that path, but couldn't find the right seam.

I'll merge this, but will continue iterating.

Thanks for reviewing!

**Pull request merged**